GliaWidgets/Sources/Coordinators/Chat/ChatCoordinator.swift

```diff
@@ -32,6 +32,13 @@ class ChatCoordinator: SubFlowCoordinator, FlowCoordinator {
     private(set) var quickLookController: QuickLookController?
     private let environment: Environment
     private let startWithSecureTranscriptFlow: Bool
+
+    // Used to determine if handling Transferred SC should be skipped.
```

Comment:
I'd add one more slash to make view it as documentations instead of comment.

Reply:
Fixed e67aa01d7d3b1ca96fdec4d5fa1b9305c678365c